executor.go

```diff
@@ -402,6 +412,32 @@ func (e *Executor) getVerifiedFlagKey(taskName string, flagKey string) (string,
 	return flagKey, errors.New(fmt.Sprintf("Unsupported flag: %s", flagKey))
 }
 
+func (e *Executor) showTaskFlagHelpText(taskName string) {
+	taskFlags := e.taskRegistry[taskName].Flags
+	helpTemplate := template.New("helpText")
+	helpTemplate = helpTemplate.Funcs(template.FuncMap{
+		"rtos": func(r rune) string { return string(r) },
+	})
+	helpTemplate, err := helpTemplate.Parse(helpMsgTemplate)
+	if err != nil {
+		// Handle error
+		fmt.Printf("There was an error parsing the template: %s", err)
+	}
+
+	err = helpTemplate.Execute(os.Stdout, struct {
+		TaskName string
+		Flags    []TaskFlag
+	}{
+		TaskName: taskName,
+		Flags:    taskFlags,
+	})
+	if err != nil {
+		// Handle error
+		fmt.Printf("There was an error executing the template: %s", err)
```

Comment:
I don't think we should necessarily panic here; not sure how we should handle the user experience here though; maybe keeping it as a Printf is ok?